### ISSUE TITLE
Unable to get Basic example to work

### ISSUE THREAD

**Issue by apuello** [activescott/serverless-aws-static-file-handler#92]
### Expected behavior:
Deploy project

### Steps to reproduce the problem:
[Clone Basic example](https://github.com/activescott/serverless-aws-static-file-handler/tree/main/examples/basic) 

### Environment: 
Dev

Please include version of serverless-aws-static-file-handler, version of serverless, version of node, and operating system name and version

## Note: I also tried with `"serverless-aws-static-file-handler": ">=3.0.1"` & `"serverless": "^2.50.0"`

```
operating system name and version: macOs Catalina - v10.15.7 
```

```
serverless-aws-static-file-handler : "serverless-aws-static-file-handler": ">=2.0.4"
```

```
serverless
Framework Core: 2.49.0 (local)
Plugin: 5.4.3
SDK: 4.2.3
Components: 3.12.0
```

```
node v14.17.2
```

```
sls deploy
------------------
sls deploy
Serverless: Running "serverless" installed locally (in service node_modules)
Serverless: To ensure safe major version upgrades ensure "frameworkVersion" setting in service configuration (recommended setup: "frameworkVersion: ^2.49.0")

Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command config:tabcompletion
Serverless: Load command config:tabcompletion:install
Serverless: Load command config:tabcompletion:uninstall
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command upgrade
Serverless: Load command uninstall
Serverless: Load command deploy
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Load command output
Serverless: Load command output:get
Serverless: Load command output:list
Serverless: Load command param
Serverless: Load command param:get
Serverless: Load command param:list
Serverless: Load command studio
Serverless: Skipping variables resolution with old resolver (new resolver reported no more variables to resolve)
Serverless: Invoke deploy
Serverless: Warning: The command you entered did not catch on any hooks
```

**Comment by activescott** [comment]
I've been on vaca and am just now investigating this. Odd output when running serverless with debug mode:

```
SLS_DEBUG=* ./node_modules/.bin/serverless deploy --aws-profile serverless

Serverless: To ensure safe major version upgrades ensure "frameworkVersion" setting in service configuration (recommended setup: "frameworkVersion: ^2.52.1")

Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command config:tabcompletion
Serverless: Load command config:tabcompletion:install
Serverless: Load command config:tabcompletion:uninstall
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command upgrade
Serverless: Load command uninstall
Serverless: Load command deploy
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Load command output
Serverless: Load command output:get
Serverless: Load command output:list
Serverless: Load command param
Serverless: Load command param:get
Serverless: Load command param:list
Serverless: Load command studio
Serverless: Skipping variables resolution with old resolver (new resolver reported no more variables to resolve)
Serverless: Invoke deploy
Serverless: Warning: The command you entered did not catch on any hooks
```

Note the last line of `Serverless: Warning: The command you entered did not catch on any hooks`

I'll keep looking here for a bit...

**Comment by activescott** [comment]
Looks like if I downgrade serverless-aws-static-file-handler it works, but [this change in v2.0.9 to upgrade to serverless     from `^1.60.5` to `^2.33.1`](https://github.com/activescott/serverless-aws-static-file-handler/compare/v2.0.8...v2.0.9#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L42), seems to have broken things. [Apparently this is happening to other plugins](https://github.com/serverless/serverless/issues/2649). 🤦 

So a workaround should be to use version v2.0.8 instead. Confirming that now...

**Comment by activescott** [comment]
@apuello Thanks for reporting. I have discovered the root of the problem is that the plugin isn't actually compatible with serverless v2.x. In my crude testing while upgrading dependencies I obviously missed this. I have downgraded the examples for now so they should work if you pull them from the `main` branch, but I'll be deprecating the  npm version(s) of the plugin that don't work and investigating a more permanent solution to get the plugin to work with serverless v2.x soon too. I will keep this issue open until I have a resolution to that (or confirm that it isn't feasible).
